packages/story-editor/src/components/floatingMenu/elements/fontSize.js

```diff
@@ -17,20 +17,51 @@
 /**
  * External dependencies
  */
-import { Icons } from '@googleforcreators/design-system';
 import { __ } from '@googleforcreators/i18n';
+import { NumericInput } from '@googleforcreators/design-system';
```

Comment:
Note that I have generalized the numeric input for the design menu in #10754, which hasn't been merged yet. That'll partially simplify this work too.

Reply:
#10754 has just been merged, FYI.

Reply:
So l pulled that in and that input seems a bit wide for this use case. I asked @agingoldseco for thoughts.
![Screen Shot 2022-03-03 at 2 31 22 PM](https://user-images.githubusercontent.com/1820266/156657081-448d5a02-541f-45d8-b44d-b4d5ad94461b.png)
![Screen Shot 2022-03-03 at 2 28 24 PM](https://user-images.githubusercontent.com/1820266/156657085-674793e1-cddf-4e97-a4ed-ccc3d91b547a.png)

Reply:
![Screen Shot 2022-03-03 at 2 41 34 PM](https://user-images.githubusercontent.com/1820266/156657442-c89dcdfd-f046-4d55-9a27-158399864d0f.png)

Reply:
Ticket created #10799